apps/condo/domains/ai/tasks/index.js

```diff
@@ -0,0 +1,8 @@
+const { createTask } = require('@open-condo/keystone/tasks')
+
+const { executeAIFlow } = require('./executeAIFlow')
+
+
+module.exports = {
+    executeAIFlow: createTask('executeAIFlow', executeAIFlow, 'low'),
```

Comment:
I'm not sure about low priority here.. 

Usually, user would be sitting and awaiting for this flow to finish after a certain input (rewrite a comment) maybe we should use higher priority here?

Reply:
I'll leave the default value